mangaki/mangaki/settings.py

```diff
@@ -69,7 +69,11 @@
     'bootstrap4',
     'django_js_reverse',
     'rest_framework',
-    'django_celery_beat'
+    'django_celery_beat',
+    'massmailer',
+    'crispy_forms',
+    'django_bootstrap_breadcrumbs',
+    'reversion',
```

Comment:
What is reversion?

Reply:
A dependency of massmailer that is used to store past versions of the templates and queries: https://github.com/etianen/django-reversion